src/commands/birthday/scheduler.ts

```diff
@@ -0,0 +1,85 @@
+import cron, { CronJob } from 'cron';
+import * as Discord from '../../Discord';
+import { CreateMessage } from '../../Discord/message';
+import ExecutionContext from '../../ExecutionContext';
+import TriggerContext from '../../TriggerContext';
+import { Collection, Config, BirthdaysConfig } from '../../database';
+import { calculateDayOfYear } from '../../commands/birthday/_common';
+
+let job: CronJob;
```

Comment:
Instead of using global variables, can we employ dependency injection (i.e., a shared CronJob object passed from the entry point file)?